spec.bs

```diff
@@ -2941,6 +2949,189 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}

```

Comment:
Hmm, if it weren't for all the fetch() patching, I think that might make sense? Some of this might get a bit restructured later so we can reuse some of the fetch patching with other adAuctionHeaders features like the bidding and auction server and negative targeting APIs -- but, not knowing the exact specifics of how those bits will be shared, I think it makes sense to save that for another PR.